app/code/Magento/Catalog/Block/Product/ProductList/Toolbar.php

```diff
@@ -308,7 +308,8 @@ public function getCurrentDirection()
         }
 
         $directions = ['asc', 'desc'];
-        $dir = strtolower($this->toolbarMemorizer->getDirection());
+        $dir = $this->toolbarMemorizer->getDirection() ? strtolower($this->toolbarMemorizer->getDirection()) : '';
```

Comment:
```suggestion
        $dir = strtolower($this->toolbarMemorizer->getDirection() ?: '');
```

Reply:
Hello @ihor-sviziev 
Thank you for your review.
I think this is correct solution.
Please take a look comment above.